capnpc-go/templates/interfaceClient

```diff
@@ -66,6 +66,14 @@ func (c {{$.Node.Name}}) IsValid() bool {
 	return capnp.Client(c).IsValid()
 }
 
+// IsSame reports whether c and c2 refer to a capability created by the
```

Comment:
You named the parameter `other` below, so this probably should use that name instead of `c2`.

Reply:
d'oh!